run_benchmark.sh

```
@@ -16,15 +17,30 @@ fi
 
 docker compose build client
 
+# Timing JSON
+TIMING_JSON=""
+
+# Signature
 source envs/client.env.sh
```

Shellcheck (0.11.0)

[info] Not following: envs/client.env.sh was not specified as input (see shellcheck -x).

(SC1091)